src/chains.jl

```diff
@@ -333,6 +333,94 @@ Base.last(c::Chains) = last(c.value[Axis{:iter}].val)
 
 Base.convert(::Type{Array}, chn::Chains) = convert(Array, chn.value)
 
+# Convenience functions to handle different types of 
+# timestamps.
+to_datetime_vec(ts::DateTime) = [ts]
+to_datetime_vec(ts::Vector{DateTime}) = ts
+to_datetime_vec(ts::Float64) = [unix2datetime(ts)]
+to_datetime_vec(ts::Vector{Float64}) = unix2datetime.(ts)
+function to_datetime_vec(ts)
+    @warn "Timestamp type $(typeof(ts)) unknown."
+    return missing
+end
+
+"""
+    start_time(c::Chains)
+
+Retrieve a vector of start times (as `DateTime`)
+from `chain.info`. Assumes that either
+`DateTime` or unix timestamps (`Float64`) are stored in
+`chain.info.start_time`
+"""
+function start_time(c::Chains)
+    if :start_time in keys(c.info)
+        # We've got some times, print them out.
+        ts = c.info.start_time
+        return to_datetime_vec(ts)
+    else
+        # Times not found -- spit out missing.
+        return missing
+    end
+end
+
+function min_start(c::Chains)
+    ts = start_time(c)
+
+    if ts === missing
+        return missing
+    else
+        return minimum(ts)
+    end
+end
+
+"""
+    stop_time(c::Chains)
+
+Retrieve a vector of stop times (as `DateTime`)
+from `chain.info`. Assumes that either
+`DateTime` or unix timestamps (`Float64`) are stored in
+`chain.info.stop_time`
+"""
+function stop_time(c::Chains)
+    if :stop_time in keys(c.info)
+        # We've got some times, print them out.
+        ts = c.info.stop_time
+        return to_datetime_vec(ts)
+    else
+        # Times not found -- spit out missing.
+        return missing
+    end
+end
+
+function max_stop(c::Chains)
+    ts = stop_time(c)
+
+    if ts === missing
+        return missing
+    else
+        return maximum(ts)
+    end
+end
```

Comment:
It seems one only needs `min_start` and `max_stop`. Therefore I suggest, in addition to the suggestions above:

```suggestion
"""
    min_start(c::Chains)

Retrieve the minimum of the start times (as `DateTime`) from `chain.info`.

It is assumed that the start times are stored in `chain.info.start_time` as
`DateTime` or unix timestamps of type `Float64`.
"""
function min_start(c::Chains)
    return if :start_time in keys(c.info)
        # We've got some times, return the minimum.
        min_datetime(c.info.start_time)
    else
        # Times not found -- spit out missing.
        missing
    end
end

"""
    max_stop(c::Chains)

Retrieve the maximum of the stop times (as `DateTime`) from `chain.info`.

It is assumed that the start times are stored in `chain.info.stop_time` as
`DateTime` or unix timestamps of type `Float64`.
"""
function max_stop(c::Chains)
    return if :stop_time in keys(c.info)
        # We've got some times, return the minimum.
        max_datetime(c.info.stop_time)
    else
        # Times not found -- spit out missing.
        missing
    end
end
```